clang/lib/Sema/SemaOverload.cpp

```diff
@@ -11846,19 +11854,21 @@ static void DiagnoseFailedEnableIfAttr(Sema &S, OverloadCandidate *Cand) {
 }
 
 static void DiagnoseFailedExplicitSpec(Sema &S, OverloadCandidate *Cand) {
-  ExplicitSpecifier ES = ExplicitSpecifier::getFromDecl(Cand->Function);
+  assert(Cand->Function && "Candidate must be a function");
+  Function *CandFunc = Cand->Function;
```

Comment:
```suggestion
  FunctionDecl *CandFunc = Cand->Function;
```
Whoops missed this the first time around, looks like a typo, please fix.